index.html

```diff
@@ -16,7 +16,7 @@
 	</head>
 	<body>
 		<nav class="prompt">
-
+      <header class="prompt__header js-header"></header>
```

Comment:
I just checked in Atom and it looks fine too. I also noticed the formatting for the submit button looks off too on Github but fine in Atom and Vim. I'm think it's an encoding issue that's outside of our control